File: send_sms.py
# Download the helper library from https://www.twilio.com/docs/python/install
from twilio.rest import Client


# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = ''
auth_token = ''
client = Client(account_sid, auth_token)

message = client.messages \
                .create(
                     body="Add my contact",
                     from_='+15122702303',
                     media_url=['https://ebizcard.netlify.com/dan/dan_h.vcf'],
                     to='+13368800967'
                 )

print(message.sid)

File: send_sms.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once './vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACb6c7f7ac0d0f2746036da52a1b144bcf";
$token  = "905e26813dfab5c36ed3ef5cdf37d26f";
$twilio = new Client($sid, $token);

$message = $twilio->messages
                  ->create("+13368800967", // to
                           [
                               "body" => "This is the ship that made the Kessel Run in fourteen parsecs?",
                               "from" => "+14692085882",
															 "mediaUrl" => ["https://ebizcard.netlify.com/dan/dan_h.vcf"]
                           ]
                  );

print($message->sid);
